src/_P027_INA219.ino

```diff
@@ -61,8 +61,8 @@ boolean Plugin_027(uint8_t function, struct EventStruct *event, String& string)
     case PLUGIN_I2C_HAS_ADDRESS:
     case PLUGIN_WEBFORM_SHOW_I2C_PARAMS:
     {
-      const uint8_t i2cAddressValues[] = { INA219_ADDRESS,   INA219_ADDRESS2,   INA219_ADDRESS3,   INA219_ADDRESS4,
-                                           INA219_ADDRESS5,  INA219_ADDRESS6,   INA219_ADDRESS7,   INA219_ADDRESS8,
+      const uint8_t i2cAddressValues[] = { INA219_ADDRESS,   INA219_ADDRESS2,   INA219_ADDRESS5,   INA219_ADDRESS6,
+                                           INA219_ADDRESS3,  INA219_ADDRESS4,   INA219_ADDRESS7,   INA219_ADDRESS8,
```

Comment:
Moah, this makes the nrs appear in the same order, but does raise some eyebrows when reading the code....

Do you still need the defines?
Better have the nrs and the comments from the .h file in this array of address values.